recipe/build.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 mkdir build
 cd build
-cmake -DBUILD_XTP=ON -DCMAKE_INSTALL_PREFIX=${PREFIX} -DCMAKE_INSTALL_LIBDIR=lib -DINJECT_MARCH_NATIVE=OFF -DENABLE_TESTING=ON ..
+cmake -DBUILD_XTP=ON -DCMAKE_INSTALL_PREFIX=${PREFIX} -DCMAKE_INSTALL_LIBDIR=lib -DGROMACS_INCLUDE_DIR=${PREFIX}/include -DGROMACS_LIBRARY=${PREFIX}/lib/libgromacs.so -DINJECT_MARCH_NATIVE=OFF -DENABLE_TESTING=ON ..
```

Comment:
I am not sure `-DGROMACS_*` are actually needed....

Reply:
Maybe we can add that in https://github.com/conda-forge/gromacs-feedstock/pull/10.